### ISSUE TITLE
Crime Crackers (NTSC-J): Hangs at loading first boss

### ISSUE THREAD

**Issue by geoffsmiller** [MiSTer-devel/PSX_MiSTer#181]
Issue:
When you are about to enter the first boss area in Crime Crackers, after an "Are you sure?" prompt to continue, the game hangs.
![vlcsnap-2022-08-14-18h02m15s590](https://user-images.githubusercontent.com/3672075/184562027-db69afeb-6340-4c7d-8dfa-a3b0637cf2f2.png)

Reproduce:
Load the attached save state, which drops you right in front of the door. Hit the circle button by the door, then the top option in the prompt. Music, etc. continues, but nothing happens.

Save State:
[Crime Crackers (Japan)_2.zip](https://github.com/MiSTer-devel/PSX_MiSTer/files/9334499/Crime.Crackers.Japan._2.zip)

Workaround:
Once the game has hung, open the OSD and reload the CHD. Once you exit the OSD, it will continue working.

Here's a video showing the problem and the workaround. (Note: I was also experimenting with the CD open/close lid feature to see if that would also work.)

https://user-images.githubusercontent.com/3672075/184562179-0e5405f1-3ac0-4f15-b4b0-066d2934ee38.mp4

BIOS:
SCPH-5500 (I think)

CD Image:
Redump CHD (also tried bin/cue, same results)

Core Version:
PSX_20220813

**Comment by Kuba-J** [comment]
Confirmed , the game stop , but does not crash , reads continuously from the disc

**Comment by Kuba-J** [comment]
the issue is fixed, but the old save posted here doesn't work.

**Comment by geoffsmiller** [comment]
I'm using the [PSX_unstable_20221106_14b3e9.rbf](https://github.com/MiSTer-unstable-nightlies/PSX_MiSTer/releases/download/unstable-builds/PSX_unstable_20221106_14b3e9.rbf) build. Let me know if I should be using a different build.

When I use the savestate that @Kuba-J provided, the bug is fixed.

I also tried to verify starting a new game from scratch (it only takes ~10 minutes to get to the where the bug happens). The first time I tried this, I had almost the exact same problem as before. The same workaround worked, but there was some graphical corruption in the 2D UI. Unfortunately, I didn't capture this footage.

I tried twice more starting a new game from scratch, and it worked fine both times. I think whatever happened the first time can be attributed to MiSTer funkiness. I did do a cold reboot between attempts, so maybe that was all it needed. I'm documenting it here just in case there actually is some intermittent bug like a race condition. I consider this fixed, though.

Thank you for fixing this issue! fwiw, I played through the whole game recently via this core, and I found that I only hit variations of that same original bug that could be fixed with the same workaround. So I think this game should work fine start-to-finish now.
